app/views/decidim/verifications/authorizations/new.html.erb

```diff
@@ -0,0 +1,35 @@
+<main class="wrapper">
+  <div class="row collapse">
+    <div class="row collapse">
+      <div class="columns large-8 large-centered text-center page-title">
+        <h1><%= t(".authorize_with", authorizer: t("#{handler.handler_name}.name", scope: "decidim.authorization_handlers")) %></h1>
+      </div>
+    </div>
+
+    <div class="row">
+      <div class="columns large-6 medium-centered">
+        <div class="card">
+          <div class="card__content">
+            <%= authorization_form_for(handler, url: authorizations_path(redirect_url: params[:redirect_url])) do |form| %>
+              <%#  We are overwritting these file to comment the line below %>
```

Comment:
Do we need to do that ? why not show form errors?

Reply:
Yes, we need it
As we have done in other verifiers, it is necessary because we have the problem that errors are shown duplicated one over the other. (see attached image)

![screenshot-localhost-3000-2019 03 18-09-32-00](https://user-images.githubusercontent.com/28536082/54517039-eccc2280-4960-11e9-802d-ea3747cdea57.png)

This is because the helper of the verifier form, 'authorization_form_for' inherits from 'decidim_form_for' and this one, already has its way of rendering the errors with the method 'base_error_messages'
Check: https://github.com/decidim/decidim/blob/7372946d74f88820a3ff710373f2fab6460ca435/decidim-core/app/helpers/decidim/authorization_form_helper.rb#L15

and

https://github.com/decidim/decidim/blob/7372946d74f88820a3ff710373f2fab6460ca435/decidim-core/app/helpers/decidim/decidim_form_helper.rb#L21

Reply:
Isn't this a bug we have to open in Meta?

Reply:
We should